fdbclient/CoordinationInterface.h

```diff
@@ -236,4 +239,30 @@ struct ProtocolInfoRequest {
 	}
 };
 
+// Returns true if the cluster descriptor may be modified.
+struct CheckDescriptorMutableReply {
+	constexpr static FileIdentifier file_identifier = 7784299;
+	CheckDescriptorMutableReply() = default;
+	explicit CheckDescriptorMutableReply(bool value) : value(value) {}
+	bool value;
+	template <class Ar>
+	void serialize(Ar& ar) {
+		serializer(ar, value);
+	}
+};
+
+// Allows client to check if allowed to change the cluster descriptor.
+struct CheckDescriptorMutable {
+	constexpr static FileIdentifier file_identifier = 214729;
+	Key key;
+	ReplyPromise<CheckDescriptorMutableReply> reply;
+	explicit CheckDescriptorMutable(Key key) : key(key) {}
```

Comment:
Is this version of the constructor used anywhere?

Reply:
removed